### ISSUE TITLE
Figure out how to display overridden eTLD

### ISSUE THREAD

**Issue by cooperq** [EFForg/privacybadgerfirefox#389]
When users make an override on the options page, they override teh settings for the ETLD+1. This can be confusing because in the popup the fqdn is shown but cant be undone because the rule is for the ETLD.  There are two possible solutions for this.

1. Better but harder:  display the etld at the top and the fqdns slightly indented under it. ETLD gets the undo button, fqdns do not. 
2. Easier but worse UI: grey out the undo button for an fqdn whos tld is overridden, explain in the rollover text.

If you are taking this issue it is up to you which way you want to implement it.

**Comment by cooperq** [comment]
Current issues in this:
* needs subdomains to be flagged as usercookieblocked
* another bug, what was it?

**Comment by SwartzCr** [comment]
Texted you the info

**Comment by cooperq** [comment]
This looks great for the most part. Only one bug that I have found: 
If I set an individual subdomain to red and then the top level domain to yellow, I see that the individual subdomain gets put under the etld as cookie blocked, it should be blocked. Similarly if I have a top level domain that is set to green, any subdomains which I previously had set to yellow or red get put under the green top level domain. 
So it seems like etld/green trumps subdomain red and yellow, etld yellow trumps subdomain red and etld red trumps no subdomain.
I feel like the correct behavior here is that if a subdomain is already in a  user list it should not be grouped under it's top level domain. (An alternative argument could be made that it should always be grouped under it's parent domain, either way it should be consistent.)
